metricbeat/etc/beat.yml

```diff
@@ -43,4 +43,6 @@ metricbeat.modules:
   period: 10s
   processes: ['.*']
 
+  # if true, exports the CPU usage in ticks, together with the percentage values
```

Comment:
I believe I got these changes when merging in the latest master the other day into my branch.